week4_Isotopes.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "Isotopes Fractionation"
+subtitle: "Isoptopes are isoDOPE"
+author: "Kevin Rozmiarek"
+date: "Last knitted `r format(Sys.Date(), '%d %b %Y')`"
+output:
+  html_document: 
+    df_print: paged
+    number_sections: yes
+    toc: yes
+    toc_float: true
+    toc_depth: 3
+    code_folding: show
+editor_options:
+  chunk_output_type: inline
+---
+
+# Learning Goals
+
+1. Understand the role of **fractionation factors** in separating isotopes in different reservoirs
+
+2. Understand the difference between **equilibrium isotope fractionation** and **kinetic isotopic fractionation**
+
+3. Be able to numerically demonstrate the isotopic evolution of a water reservoir as a consequence of evaporation
+
+4. Be able to manipulate and deal with bad input data.
+
+# Prerequisites
+
+1. Have an understanding of what **isotopes** are and how to express them in different notations.
+
+2. Basic understanding of **R** programming, flow control, graphing, and tibble manipulation.
+
+# Background
+
+Because we just finished our lesson on what isotopes are and their notation *cough*, it is time to talk about **isotope fractionation**. **Isotope fractionation** is the separation or partitioning of a reservoir(s) stable isotopes due to a processe that use masses differently. Some processes depend on the relative differences in the masses of different isotopes in chemical equilibrium (**equilibrium isotopic fractionation**). Other processes depend on rate-dependent transfers or site-specific chemical binding effects (**kinetic isotopic fractionation**).
```

Comment:
I think you did a really good job explaining equilibrium vs kinetic fractionation without going off the deep end!